doc/classes/Array.xml

```diff
@@ -138,18 +127,19 @@
 			<param index="0" name="method" type="Callable" />
 			<description>
 				Calls the provided [Callable] on each element in the array and returns [code]true[/code] if the [Callable] returns [code]true[/code] for [i]all[/i] elements in the array. If the [Callable] returns [code]false[/code] for one array element or more, this method returns [code]false[/code].
-				The callable's method should take one [Variant] parameter (the current array element) and return a boolean value.
+				The [param method] should take one [Variant] parameter (the current array element) and return a boolean value.
```

Comment:
All of the methods that accept Callables have this problem where their wording to describe the Callable parameter is inconsistent, but I am not sure exactly what the ideal way to do it would be.

It doesn't help that currently some of these parameters are called "method" but I digress for now. That can be addressed another time.

Reply:
In reality it would be a good idea to move what the Callable should *be* to the second paragraph. That is the main inconsistency issue.